metadata.rb

```diff
@@ -2,14 +2,14 @@
 maintainer       'Oregon State University'
 maintainer_email 'chef@osuosl.org'
 license          'Apache-2.0'
-chef_version     '>= 14.0'
+chef_version     '>= 16.0'
 issues_url       'https://github.com/osuosl-cookbooks/osl-keepalived/issues'
 source_url       'https://github.com/osuosl-cookbooks/osl-keepalived'
 description      'Installs/Configures osl-keepalived'
-version          '1.0.2'
+version          '0.4.0'
```

Comment:
Something is wrong here, this shouldn't be changed. (same below).

Reply:
idk what happened here, my script must have screwed up